lib/matplotlib/tests/test_backend_webagg.py

```diff
@@ -0,0 +1,22 @@
+import subprocess
+import sys
+import pytest
+
+
+@pytest.mark.parametrize("backend", ["webagg", "nbagg"])
+def test_webagg_fallback(backend):
+    if backend == "nbagg":
+        pytest.importorskip("IPython")
+    test_code = (
+        "import os;"
+        + f"os.environ['MPLBACKEND'] = '{backend}';"
+        + "import matplotlib.pyplot as plt; "
+        + "print(plt.get_backend());"
+        f"assert '{backend}' == plt.get_backend().lower();"
+    )
+    ret = subprocess.call(
+        [sys.executable, "-c", test_code],
+        env={"MPLBACKEND": backend, "DISPLAY": ""}
```

Comment:
this line, even just `env={"MPLBACKEND": backend}` is what's triggering the windows failures